spec/features/user_email_is_valid_on_sign_up_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.feature "Sign-up email", type: :feature do
+  scenario "User email has correct format" do
+    visit "/users/sign_up"
+    fill_in "user_email", with: "example"
+    fill_in "user_password", with: "1234567"
+    fill_in "user_password_confirmation", with: "1234567"
+    click_button "Sign up"
+    expect(current_path).to eq("/users")
+    expect(page).to_not have_content("Welcome! You have signed up successfully.")
+  end
+
+  scenario "User email is unique" do
```

Comment:
You can now use the helper method to specify what the email is of the first user you sign in to show that it is the same as the second